controllers/apply/under10k/fields/project-dates-next.js

```diff
@@ -58,6 +59,9 @@ module.exports = {
                 cy: `Pryd hoffech chi gael yr arian os ydych chi'n cael eich dyfarnu?`,
             }),
             options: options(),
+            schema:  Joi.string().valid(
+                options().filter((option) => !has('disabled')(option.attributes)).map((option) => option.value)
```

Comment:
There's quite a lot going on in one line here, maybe we could make these named functions to make it clearer what's happening? Something like:

```
    get schema() {
        const enabledOptions = (option) => !has('disabled')(option.attributes);
        const optionValues = (option) => option.value;
        return options().filter(enabledOptions).map(optionValues);
    }
```